v1.2-client.yml

```diff
@@ -1217,35 +1227,74 @@ definitions:
       gender:
         type: string
         x-nullable: true
+        x-validation: true
+        enum:
+        - "M"
+        - "F"
+        - ""
+      google_apps:
+        $ref: "#/definitions/GoogleApps"
+        x-nullable: true
       grade:
         type: string
         x-nullable: true
+        x-validation: true
+        enum:
+        - "1"
+        - "2"
+        - "3"
+        - "4"
+        - "5"
+        - "6"
+        - "7"
+        - "8"
+        - "9"
+        - "10"
+        - "11"
+        - "12"
+        - "PreKindergarten"
+        - "Kindergarten"
+        - "PostGraduate"
+        - "Other"
       hispanic_ethnicity:
         type: string
         x-nullable: true
-      iep_status:
-        type: string
-        x-nullable: true
```

Comment:
We've removed this and it might still come through in the client! Also are there other protected fields we're missing?